src/plugins/intel_cpu/src/nodes/executors/acl/acl_transpose.hpp

```diff
@@ -45,6 +45,10 @@ class ACLTransposeExecutorBuilder : public TransposeExecutorBuilder {
                       srcDescs[0]->getShape().getRank());
             return false;
         }
+        if (srcDescs[0]->getPrecision() == InferenceEngine::Precision::FP16) {
```

Comment:
1. What about I8 or I64? Do they have optimized impls? If not lets enable it only for fp32
2. Seems like we also have to check that in and out precisions are the same. I suspect ACL doesn't support different in/out precisions.